dotnet/src/Connectors/Connectors.Onnx/OnnxRuntimeGenAIPromptExecutionSettings.cs

```diff
@@ -28,10 +32,34 @@ public static OnnxRuntimeGenAIPromptExecutionSettings FromExecutionSettings(Prom
             return settings;
         }
 
-        var json = JsonSerializer.Serialize(executionSettings);
+        var json = JsonSerializer.Serialize<object>(executionSettings);
```

Comment:
Why is this being serialized as object? Seems like the type is known and you might have been able to avoid the annotation in the method.